automation/deploy_igz/README.md

```diff
@@ -0,0 +1,10 @@
+# Deploy mlrun-api from your current code on a live system (for debugging)
+
+
+In order to deploy your current code (for debugging), you need the following:
+
+* Install automation/requirements.txt
+* Create a deploy_env.yaml based on deploy_env_template.yaml
```

Comment:
1. I suggest to add `deploy_env.yaml` to a `.gitignore` file
2. Note that the template file and the code use `.yml`, not ~~`.yaml`~~

Reply:
good point. done